_visualisation/optional/metadata.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Metadata Overlays
+layout: visualisation
+---
+
+Metadata Overlays is an [Optional Feature](/visualisation/optional/) of BOVS Diagrams. It allows further information about Parties to be shown as icons overlaid on their Nodes.
+{: .lead}
+
+
+## Metadata Types
+
+The types of metadata you wish to show are your choice; we envisage that these are likely to include properties such as:
+
+| Metadata Type              | Explanation
+| ----------------------------
+| Regime of Jurisdiction     | Which Regime (e.g. country) has jurisdiction over this Party?
+| Disclosure Status          | Has this Party fully disclosed its beneficial ownership information?
+| Below Disclosure Threshold | Is this Party below the threshold for disclosure?
+{: .unstriped }
+
+Each metadata type must be a property of a Party that can take on one or more of a finite list of values.
+
+
+## Appearance
+
+Metadata values are shown by overlaying icons around the circumference of the related Party's Node.
+
+For each type of metadata, determine all of its possible values, and what icons will be used to indicate each of these values.
+
+Each type of metadata has a reserved position where its related icons can appear. You may determine these positions as you see fit, however we recommend:
+
+* Prefer positions at 45, 135, 225 and 315 degress (i.e. the diagonal compass points)
+* Avoid positions on the bottom of the circle, where icons may appear too close to the Node's label.
+
+When a particular value for a type of metadata applies to a Party, show the icon for that value in the position reserved for that type of metadata.
+
+### Null Values
+
+Where a type of metadata can have a value of **false, none, zero or not applicable**:
+
+* You may use an explicit icon for this.
+* You may denote this simply by *not* showing an icon in these cases.
+
+### Unknown Values
+
+It is possible that the value for a type of metadata is unknown for some Parties. In this case, show a question mark in place of an icon, in the correct position for that type of metadata.
+
+### Key
+
+Include a key on your diagram fully explaining all of your icons.
+
+
+## Relevance
+
+Show metadata icons on all and only the [relevant](/visualisation/core/relevance) Nodes in your diagram.
+
+* Metadata should not be added to non-relevant Nodes.
+* Omission of metadata (on a relevant Node) is meaningful. This either means:
+  * Compared with when the icon is shown, the opposite is true.
+  * Zero, none or not applicable.
+
+
+## Multiple Metadata
+
+If there are multiple values for a type of metadata:
+
+* Show multiple icons.
+* Cluster the icons together by slightly overlapping them in a line extending from the Node.
+
+Selected icons can be grouped into a Stack. Do this if too many types of metadata apply.
```

Comment:
Stack could be linked here

Reply:
Strictly it is not a reference to the Stack rule which applies to Nodes... it is just a rule that operates similarly. That consistency is hopefully helpful, but it might be confusing to make the direct link?

Reply:
Ah sorry, I was obviously reading too quickly. Agree it would be confusing to link to that. Maybe don't capitalise it here?